packages/vuetify/src/composables/virtual.ts

```diff
@@ -37,6 +38,13 @@ export function useVirtual <T> (props: VirtualProps, items: Ref<readonly T[]>, o
     },
   })
   const containerRef = ref<HTMLElement>()
+
+  const isItemBeyondContainer = computed(() => containerRef.value && containerRef.value.clientHeight <= itemHeight.value)
```

Comment:
DOM APIs aren't reactive, there's `contentRect` for this

Reply:
changed and dynamic size items are working now